pallets/teerex/src/lib.rs

```diff
@@ -216,6 +217,7 @@ pub mod pallet {
 			let sender = ensure_signed(origin)?;
 			ensure!(ra_report.len() <= MAX_RA_REPORT_LEN, <Error<T>>::RaReportTooLong);
 			ensure!(worker_url.len() <= MAX_URL_LEN, <Error<T>>::EnclaveUrlTooLong);
+			ensure!(Self::enclave_count() < MAX_ENCLAVE_NUMBER, <Error<T>>::ExceedEnclaveNumber);
```

Comment:
`<=` ?

Reply:
But the outcome of this call is not always new enclave addition, sometimes it just updates already registered enclave

```
		let enclave_idx = if <EnclaveIndex<T>>::contains_key(sender) {
			log::info!("Updating already registered enclave");
			<EnclaveIndex<T>>::get(sender)
		}
```

Does it mean the update wouldn't be possible once we hit the limit ?

Reply:
Hi @songzhouran , could you please address this question before resolving the conversation ?

Reply:
Now already registered enclave is excluede from checking.